app/views/layouts/admin.html.erb

```diff
@@ -4,14 +4,28 @@
   <title>Onboarding</title>
   <%= csrf_meta_tags %>
 
-  <%= stylesheet_link_tag 'admin', media: 'all', 'data-turbolinks-track': 'reload' %>
-  <%= javascript_include_tag 'admin', 'data-turbolinks-track': 'reload' %>
+  <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
```

Comment:
I copied this code from the other PR, so this part is wrong here, but I realised and switched it again in the next commits.